src/controllers/hibp.js

```diff
@@ -2,9 +2,27 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+import { acceptedLanguages, negotiateLanguages } from 'fluent-langneg'
```

Comment:
hmm... don't we already store the user's language preference in db?

Reply:
Yes, you are right!

Reply:
Ah, maybe I was thinking of the server store: maybe the `getLocale ()` function in `utils/fluent.js` already does this work for you?  

It's been a while, but I _think_ we detect and negotiate lang preference on request already, and then store it to avoid running these functions again.  I could be mistaken, more investigation needed!